### PR TITLE
Issue #78

### PR DESCRIPTION
**Pull request by eugene-p**
Created:
 - Restfull endoint for entity type
 - Revamped migration
 - created postman collection
 - created new routes task

### CODE REVIEW

**Comment by eugene-p** [comment]
Code review please

**Comment by cedar-technologies** [comment]
Will do cr soon

File: app/core/entity/resources.py [hunk 1]
```diff
@@ -15,59 +15,54 @@
...
+    db.session.begin()
+    db.session.delete(entity_type)
+    db.session.commit()
+
+    return "Entity Type deleted", 200
+
+@blueprint.route('entity-type', methods=['POST'])
+def create_entity_type():
+    if not request.json:
+        abort(400, description='Content type not supported')
```
Comment (cedar-technologies) [inline app/core/entity/resources.py]: could return [415](https://www.w3.org/Protocols/rfc2616/rfc2616-sec10.html#sec10.4.16)

File: app/__init__.py [hunk 2]
```diff
@@ -79,9 +84,9 @@ def initialize_core(app):
     core.init_app(app)
     log.info("core initialized")
 
+
 def initialize_app(app):
     configure_app(app)
     log.info('initialize app')
-    blueprint = Blueprint('api', __name__, url_prefix='/api')
-    api.init_app(blueprint)
+#    api.init_app(blueprint)
```
Comment (cedar-technologies) [inline app/__init__.py]: commented out code

**Pull request merged**